components/cam/bld/namelist_files/use_cases/2000_scream_lr.xml

```diff
@@ -99,7 +99,10 @@
 <iradsw hgrid="ne1024np4"> 4 </iradsw>
 <iradlw hgrid="ne1024np4"> 4 </iradlw>
 
-<!-- For high-res, we use se_ftype = 1 to improve dynamics/physics coupling -->
-<se_ftype hgrid="ne1024np4">1</se_ftype>
+<!-- Use se_ftype = 1 to improve dynamics/physics coupling -->
+<se_ftype>1</se_ftype>
```

Comment:
Sadly, I think this should probably remain ftype=2 like E3SM. My thinking with ftype=1 is that ne1024 can handle dumping all physics tendencies at once because it has a 75 sec coupling frequency. ne30 probably can't handle it because it has a 30 min dt. And I know that we can run HR at lower res and LR at higher res, so my timings aren't always right... but those are the resolutions we're targeting HR and LR to and it's nice to have options stay consistent within a compset so I think this path makes sense.

Reply:
Returned to 2.

Reply:
ftype still looks like 1 to me. Is this a problem with my web browser (which I refreshed), or did you not push your changes, @brhillman ?